test/runtime-match-test.js

```diff
@@ -77,4 +77,17 @@ describe('Runtime Match', function() {
       });
     }, { block: 'b', foo: 'This is' }, 'This is ContextChild');
   });
+
+  it('should pass BEMContext instance and json to extend body',
```

Comment:
Can you move test to [test/modes-extend-test.js](https://github.com/bem/bem-xjst/blob/master/test/modes-extend-test.js)?